tests/unit/format-message-test.js

```diff
@@ -63,6 +63,17 @@ test('should throw if called with out a value', function(assert) {
     assert.throws(runAppend(view), Error, 'raised error when not value is passed to format-message');
 });
 
+test('should throw intl-get is used standalone helper', function(assert) {
+    expect(1);
+    view = this.intlBlock('{{intl-get "messages.foo.bar"}}');
+
+    try {
+        runAppend(view);
+    } catch (ex) {
+        assert.ok(ex, 'intl-get threw when run standalone');
```

Comment:
`assert.throws` doesn't pick up the Ember.assert so resorting to a good ol' try/catch + expect